### PR TITLE
pan-os-edit-rule command: Added support for removing and adding group profile settings

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6833

## Description
As in the title, just replacing was supported till now.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by GuyAfik — COMMENTED** [review]
nice work, see my comment, please add unit-test for it if possible

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -3767,35 +3767,44 @@ def panorama_edit_rule_items(rulename: str, element_to_change: str, element_valu
...
+        current_objects_items = panorama_get_current_element(element_to_change, params['xpath'])
+        if behaviour == 'add':
+            values = list((set(current_objects_items)).union(set(element_value)))
+        else:  # remove
+            values = [item for item in current_objects_items if item not in element_value]
+            if not values:
+                raise Exception(f'The object: {element_to_change} must have at least one item.')
+
+        params['element'] = add_argument_list(values, element_to_change, True)
+        result = http_request(URL, 'POST', body=params)
```
[the remark this answers is not in the document]
Comment (rshunim) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3789]: I just indented these lines, they aren't my code

**Comment by rshunim** [comment]
UT was added

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -3767,35 +3767,45 @@ def panorama_edit_rule_items(rulename: str, element_to_change: str, element_valu
...
-    else:  # remove
-        values = [item for item in current_objects_items if item not in element_value]
-        if not values:
-            raise Exception(f'The object: {element_to_change} must have at least one item.')
+    # in this case, we want to remove the profile-setting group
+    if element_to_change == 'profile-setting':
+        params['action'] = 'set'
+        params['element'] = '<profile-setting><group/></profile-setting>'
+        http_request(URL, 'POST', body=params)
+        return_results(f'Rule edited successfully.')
```
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3776]: dont you need to use `return_results` here the same way as lines 3799?

**Review by GuyAfik — APPROVED** [review]
looks good, just one last comment

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 2 above)
Comment (rshunim) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3776]: Fixed it, thanks!

**Pull request merged**